### PR TITLE
Update get-started-core-ediscovery.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated steps as options have changed

### CODE REVIEW

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9894

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/get-started-core-ediscovery.md [hunk 1]
```diff
@@ -75,17 +75,13 @@ Complete the following steps to add users to the eDiscovery Manager role group:
...
-6. Click **Add** to add the selected users, and then click **Done**.
-
-7. Click **Save** to add the users to the role group, and then click **Close** to complete the step.
+2. On the <a href="https://go.microsoft.com/fwlink/p/?linkid=2173597" target="_blank">**Permissions**</a> page, expand the **Microsoft Purview solution** section
+3. Click on **Roles** section and Click on **eDiscovery Manager** check box.
+4. On the eDiscovery Manager flyout page, Click **Edit Role Group**/
+5. On the **Choose eDiscovery Manager** page in the edit role group wizard, Click **Choose dicovery manager**
+6. Click **Add** then select the checkbox for all users you want to add to the role group.
+7. Click **Add** to add the selected users, and then click **Done**.
+8. Click **Save** to add the users to the role group, and then click **Close** to complete the step.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/get-started-core-ediscovery.md]:
```suggestion
2. On the <a href="https://go.microsoft.com/fwlink/p/?linkid=2173597" target="_blank">**Permissions**</a> page, expand the **Microsoft Purview solution** section.
3. Select **Roles** and then select the **eDiscovery Manager** check box.
4. On the eDiscovery Manager flyout page, select **Edit Role Group**.
5. On the **Choose eDiscovery Manager** page in the edit role group wizard, select **Choose discovery manager**.
6. Select **Add**, then select the checkbox for all users you want to add to the role group.
7. Select **Add** to add the selected users, and then select **Done**.
8. Select **Save** to add the users to the role group, and then select **Close** to complete the step.
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by XXLMandalorian013** [comment]
Cool, thanks guys!

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Review by XXLMandalorian013 — APPROVED** [review]

**Comment by XXLMandalorian013** [comment]
@yogkumgit  I think this still needs merged.

**Comment by yogkumgit** [comment]
@robmazz Gentle reminder.

**Comment by XXLMandalorian013** [comment]
@yogkumgit Another reminder.

**Comment by yogkumgit** [comment]
Closing as per the comment in #9894

**Pull request closed without merging**